### DISCUSSION TITLE
Incorrect facet responses for posts containing emojis #380

### DISCUSSION THREAD

**Discussion by MadhuGowthaman** [bluesky-social/atproto#3184] [Bluesky Lexicons]
Hello everyone,

For the following post, I'm getting incorrect facet responses.

Post Url: [https://bsky.app/profile/testjjzz.bsky.social/post/3lbwoz6effs2m](https://bsky.app/profile/testjjzz.bsky.social/post/3lbwoz6effs2m)

Response:

> {
> "createdAt": "2024-11-27T13:50:05.980Z",
> "langs": [
> "en"
> ],
> "text": "😀 #abcdef",
> "$type": "app.bsky.feed.post",
> "facets": [
> {
> "features": [
> {
> "tag": "abcdef",
> "$type": "app.bsky.richtext.facet#tag"
> }
> ],
> "index": {
> "byteEnd": 12,
> "byteStart": 5
> }
> }
> ]
> }

Could someone please explain how to properly utilize facet information for this post?

Thank you!

**Comment by DavidBuchanan314** [comment]
The offsets are correct, note that they're utf8 byte indexes, not codepoint indexes or anything else.

**Reply by MadhuGowthaman** [reply]
@DavidBuchanan314 Thank you for your response. I’ll review the information you’ve provided.
